app/models/concerns/declarable.rb

```diff
@@ -2,36 +2,49 @@ module Declarable
   extend ActiveSupport::Concern
   include Formatter
 
+  COMMON_UNION_MEASURE_ORDER = [
+    Sequel.desc(:measures__measure_generating_regulation_id),
+    Sequel.desc(:measures__measure_generating_regulation_role),
+    Sequel.desc(:measures__measure_type_id),
+    Sequel.desc(:measures__goods_nomenclature_sid),
+    Sequel.desc(:measures__geographical_area_id),
+    Sequel.desc(:measures__geographical_area_sid),
+    Sequel.desc(:measures__additional_code_type_id),
+    Sequel.desc(:measures__additional_code_id),
+    Sequel.desc(:measures__ordernumber),
+    Sequel.desc(:effective_start_date),
+  ].freeze
+
   included do
     one_to_many :measures, primary_key: {}, key: {}, dataset: -> {
       Measure.join(
         Measure.with_base_regulations
                .with_actual(BaseRegulation)
                .where({ measures__goods_nomenclature_sid: uptree.map(&:goods_nomenclature_sid) })
                .where { Sequel.~(measures__measure_type_id: MeasureType.excluded_measure_types) }
-               .order(Sequel.desc(:measures__measure_generating_regulation_id), Sequel.desc(:measures__measure_type_id), Sequel.desc(:measures__goods_nomenclature_sid), Sequel.desc(:measures__geographical_area_id), Sequel.desc(:measures__geographical_area_sid), Sequel.desc(:measures__additional_code_type_id), Sequel.desc(:measures__additional_code_id), Sequel.desc(:measures__ordernumber), Sequel.desc(:effective_start_date))
```

Comment:
I'm not changing these, I'm consolidating so I can make the same change in one place